requirements.in

```diff
@@ -12,7 +12,6 @@ google-auth
 google-api-python-client
 paramiko
 pathspec
-httpx
```

Comment:
This is a duplicate entry here https://github.com/getsentry/sentry-infra-tools/pull/21/files#diff-de469cc9cf8992ffaec661b4e087cfded3ec6da722072ec1fdf11f61690fd1b8L10